test/e2e/admin_portal.go

```diff
@@ -274,4 +274,128 @@ var _ = Describe("Admin Portal E2E Testing", func() {
 		}
 		Expect(elem.Text()).To(Equal(clusterName))
 	})
+
+	It("Should display refresh button to get latest details for each tab in cluster detail page", func() {
+		wd.Get(host + "/" + "?resourceid=" + resourceIDFromEnv())
+		wd.WaitWithTimeout(conditions.ElementIsLocated(selenium.ByID, "ClusterDetailPanel"), time.Second*3)
+
+		detailPanel, err := wd.FindElement(selenium.ByID, "ClusterDetailPanel")
+		if err != nil {
+			SaveScreenshotAndExit(wd, err)
+		}
+
+		Expect(detailPanel.IsDisplayed()).To(BeTrue())
+
+		wd.Wait(conditions.ElementIsLocated(selenium.ByCSSSelector, "div[aria-label='Refresh']"))
+
+		button, err := wd.FindElement(selenium.ByCSSSelector, "div[aria-label='Refresh']")
+		button.Click()
+		if err != nil {
+			SaveScreenshotAndExit(wd, err)
+		}
```

Comment:
Can we move the error checking before the `Click` event?

Reply:
done